example/resource-manager/10-crd-managedresource.yaml

```diff
@@ -124,10 +131,10 @@ spec:
                       description: The reason for the condition's last transition.
                       type: string
                     status:
-                      description: Status of the ManagedResource condition.
+                      description: Status of the condition, one of True, False, Unknown.
                       type: string
                     type:
-                      description: Type of the ManagedResource condition.
+                      description: Type of the Shoot condition.
```

Comment:
In g/grm, there was a type called `resourcesv1alpha1.ManagedResourceCondition`. This type was quasi equivalent to `gardencorev1beta1.Condition`. It wasn't used back then when GRM was introduced because we didn't want to vendor g/g. Now, since g/grm is integrated in g/g, this type was replaced in https://github.com/gardener/gardener/pull/4757/commits/ef3633aaa902617a89f0cb3531636eb76ab8c7ca in favor of the equivalent `gardencorev1beta1.Condition` type. 
You're right, the doc string is confusing, I improved it to make it more universally usable (it was already used before in other types as well).

Reply:
I understand. Thanks for explanation!